### PR TITLE
update bio

### PR DESCRIPTION
**Pull request by ocefpaf**
I'm not sure if I am modifying things in the right place :grimacing:

### CODE REVIEW

File: _data/amy.yml [hunk 1]
```diff
@@ -1292,9 +1292,9 @@ airports:
 - country: br
   instructors:
   - {name: Filipe Fernandes, user: fernandes_filipe}
-  latitude: -12.910833
-  longitude: -38.330833
-  name: Salvador de Bahia
+  latitude: -27.59
```
Comment (rgaiacs) [inline _data/amy.yml]: This info should be updated from AMY.

Comment (ocefpaf) [reply _data/amy.yml]: Thanks! I'll update the PR to reflect that and check what I can do with AMY.

Comment (jduckles) [reply _data/amy.yml]: In AMY it will be set by your airport code.

Comment (ocefpaf) [reply _data/amy.yml]: Thanks! I updated AMY a few days ago. (Sorry for my ignorance but I AMY did not existed when I took the instructor training :grimacing:)

A related question regarding AMY. I noticed that there are some fields pending on my profile, that is odd b/c I do recall doing most those back in 2014.

photo4933784363942914118

Comment (jduckles) [reply _data/amy.yml]: This is a bug experienced by older instructors who we didn't track the steps for. Can you please send a note to team@carpentries.org and the team will fix your profile for you.

Comment (ocefpaf) [reply _data/amy.yml]: Will do. thanks!

**Comment by ocefpaf** [comment]
Thanks!

**Pull request merged**